libraries/exporters/src/main/java/com/siemens/sw360/exporter/ProjectExporter.java

```diff
@@ -91,53 +113,92 @@ public int getColumns() {
         }
 
         @Override
-        public List<String> makeRow(Project project) {
-            List<String> row = new ArrayList<>(getColumns());
+        public SubTable makeRows(Project project) {
+            return extendedByReleases
```

Comment:
This class and the other exporter share much of the structure and code. Maybe this could be moved to a common parent class.